preload/preload-edge-cases.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<title>Test cases for scenarios not defined in the preload spec</title>
+<script src="/resources/testharness.js" a></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+    const serveUrl = './resources/inc.py'
+    function uuidv4() {
+        return ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c =>
+            (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
```

Comment:
https://wicg.github.io/uuid/ could've been interesting here, but I don't think it's broadly supported.